BridgeAppSDK/SBAScheduledActivityManager.swift

```diff
@@ -322,8 +322,22 @@ open class SBAScheduledActivityManager: NSObject, SBASharedInfoController, ORKTa
         return activities.find({ $0.taskIdentifier == taskIdentifier })
     }
     
+    /**
+     Delete the output directory.
+     */
+    @objc(deleteOutputDirectoryForTaskViewController:)
+    open func deleteOutputDirectory(for taskViewController: ORKTaskViewController) {
+        guard let outputDirectory = taskViewController.outputDirectory else {return}
+        do {
+            try FileManager.default.removeItem(at: outputDirectory)
+        } catch let error as NSError {
+            debugPrint("Unable to remove ResearchKit output directory at \(outputDirectory): \(error)")
```

Comment:
True. I'm never sure where the right place is to draw the line between "things we might need to troubleshoot in production apps" and "things we want to catch before it goes to production but don't want to clutter the logs with/potentially reveal sensitive info/etc. in production apps".